src/status_im/contexts/shell/share/view.cljs

```diff
@@ -1,18 +1,14 @@
 (ns status-im.contexts.shell.share.view
   (:require
-    [clojure.string :as string]
```

Comment:
The removed code in this file was moved to new file above - `share/profile/view`